ui/pages/confirm-transaction-base/confirm-transaction-base.component.js

```diff
@@ -537,19 +550,27 @@ export default class ConfirmTransactionBase extends Component {
                 }
               />
             ),
-            <TransactionDetailItem
-              key="total-item"
-              detailTitle={t('total')}
-              detailText={renderTotalDetailText()}
-              detailTotal={renderTotalDetailTotal()}
-              subTitle={t('transactionDetailGasTotalSubtitle')}
-              subText={t('editGasSubTextAmount', [
-                <b key="editGasSubTextAmountLabel">
-                  {t('editGasSubTextAmountLabel')}
-                </b>,
-                renderTotalMaxAmount(),
-              ])}
-            />,
+            isMultiLayerFeeNetwork && (
+              <MultiLayerFeeMessage
+                transaction={txData}
+                layer2fee={hexMinimumTransactionFee}
+              />
+            ),
+            !isMultiLayerFeeNetwork && (
```

Comment:
This does not change the code, only hides this component on `mutlilayerFeeNetwork`s. The multiline diff is just due to indentation changes.